<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyWidget</class>
 <widget class="QDialog" name="CopyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Copying</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Copying</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="copyLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Co&amp;pying %1 files</string>
          </property>
          <property name="buddy">
           <cstring>copyToComboBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="copyToComboBox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&amp;Existing files</string>
          </property>
          <property name="buddy">
           <cstring>comboBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Ask</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Overwrite</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Append</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Skip</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Only newer files</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="copyContentLinksCheckBox">
        <property name="text">
         <string>Copy contents of symbolic &amp;links</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="filterCheckBox">
        <property name="text">
         <string>&amp;Use filter</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="includeMaskLabel">
          <property name="text">
           <string>Include mask</string>
          </property>
          <property name="buddy">
           <cstring>includeMaskComboBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="includeMaskComboBox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Exclude mask</string>
          </property>
          <property name="buddy">
           <cstring>excludeMaskComboBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="excludeMaskComboBox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="copyPushButton">
          <property name="text">
           <string>&amp;Copy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="settingsPushButton">
          <property name="text">
           <string>&amp;Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelPushButton">
          <property name="text">
           <string>Ca&amp;ncel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="1" margin="5"/>
 <resources/>
 <connections/>
</ui>
